M2/Macaulay2/packages/VersalDeformations.m2

```diff
@@ -6,13 +6,13 @@
 ---------------------------------------------------------------------------
 newPackage("VersalDeformations",
     Headline => "versal deformations and local Hilbert schemes",
-    Version => "2.1",
-    Date => "April 10, 2020",
+    Version => "2.2",
+    Date => "October 30, 2020",
     Authors => {
         {Name => "Nathan Ilten",
 	  HomePage => "http://www.sfu.ca/~nilten/",
 	  Email => "nilten@sfu.ca"}},
-    Keywords => {"Deformation Theory"},
+--  Keywords => {"Deformation Theory"},
```

Comment:
Perhaps you commented this out so it would run with an older version of M2?

Reply:
Keywords is giving an error with version 1.16, which is the latest distributed version.

Reply:
Well, as this is the code to be incorporated into 1.17, let's leave the line in.

Or you could write an "if" statement based on testing the version number that would run newPackage both ways.  Then your source code would work with both versions.